lib/util/helpers.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var _ = require('underscore');
+
+module.exports = {
+    getPreviousSteps: function(steps, route) {
```

Comment:
These arguments seem backwards, we need possible previous steps based on a route.